bioimageio/spec/model/v0_4/raw_nodes.py

```diff
@@ -117,9 +120,9 @@ class Model(_RDF):
     run_mode: Union[_Missing, RunMode] = missing
     sample_inputs: Union[_Missing, List[Union[URI, Path]]] = missing
     sample_outputs: Union[_Missing, List[Union[URI, Path]]] = missing
-    timestamp: datetime = missing
-    type: Literal["model"] = missing
-
     test_inputs: List[Union[URI, Path]] = missing
     test_outputs: List[Union[URI, Path]] = missing
+    timestamp: datetime = missing
+    training_data: Union[_Missing, Dict[str, Union[str, Dataset]]] = missing
```

Comment:
I think that we haven't fully settled on whether we want a `List` or a `Dict` here yet. Let's finalize that decision in the meeting tomorrow.

Reply:
as discussed in short meeting with @constantinpape @oeway : no list nor dict. single dataset id (or in-place RDF) and we'll take it from there when need be...